buttons-examples/iOS/ios-FAB.md

```diff
@@ -152,80 +152,204 @@ in the spec.
 
+## Regular FABs
 
-## Floating action button
+Regular FABs are FABs that are not expanded and are a regular size.
 
-You can apply a theme to the FAB using `Themes`.
+### Regular FABs example
 
-Source Code APIs:
+To create a regular FAB use the `+floatingButtonWithShape:` constructor with a value of `MDCFloatingButtonShapeDefault` and make sure the `mode` property is set to `MDCFloatingButtonModeNormal`.
 
-* MDCButton  (a subclass of [UIButton](https://developer.apple.com/documentation/uikit/uibutton))
-    * [Class description](https://material.io/develop/ios/components/buttons/api-docs/Classes/MDCButton.html)
-    * [GitHub source](https://github.com/material-components/material-components-ios/blob/develop/components/Buttons/src/MDCButton.h)
-* [Themes class description](https://material.io/develop/ios/components/theming/)  <!-- This is slated to be deprected, though the examples from https://material.io/develop/ios/components/buttons/api-docs/Classes/MDCButton.html appear to use this class--> 
+For more information on theming FABs see the [Theming section](#theming). 
 
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+let fab = MDCFloatingButton(shape: `default`)
+```
 
-<!-- To do:
-* add example code
-* add example code render screenshot
+#### Objective-C
+```objc
+MDCFloatingButton *fab =
+    [MDCFloatingButton floatingButtonWithShape:MDCFloatingButtonShapeDefault];
+```
+<!--</div>-->
 
 ### Anatomy and key properties
 
-A FAB has a container and an icon.
+A regular FAB has a container and an icon.
+
+1. Container
+1. Icon
+
+#### Container attributes
+
+&nbsp;           | Attribute             | Related method(s)                                                            | Default value
+---------------- | --------------------- | ---------------------------------------------------------------------------- | -------------
+**Color**        | `backgroundColor`  | `setBackgroundColor:forState:`<br/>`backgroundColorForState`<br/> | blue 500 from this [Theming doc](https://material.io/go/design-color-theming#color-color-palette).
+**Stroke color** |  | `setBorderColor:forState:`<br/>`borderColorForState:` | `nil`
+**Stroke width** |  | `setBorderWidth:forState:`<br/>`borderWidthForState:` | `0`
+**Ripple color** | `inkColor`   | `setInkColor`<br/>`inkColor` | White at 20% opacity
+
+#### Icon attributes
+
+&nbsp;                                          | Attribute         | Related method(s)                                         | Default value
+----------------------------------------------- | ----------------- | --------------------------------------------------------- | -------------
+**Icon**                                        | `imageView`        | `setImage:forState:`<br/>`imageForState:` | `nil`
+**Color**                                       | `imageView.tintColor` | `setImageViewTintColor:forState:`<br/>`imageViewTintColorForState:` | `nil`
 
-![FAB anatomy diagram](assets/FAB-anatomy.png)
```

Comment:
Does the iOS FAB not follow the anatomy diagram in any way?

Reply:
I'm not sure what you mean

Reply:
Is there a reason why you deleted the line for the FAB anatomy diagram?

Reply:
I got rid of the markdown and replaced it with the html image so I could specify the height because it looks too big otherwise. This is what it looks like with the preexisting `![FAB anatomy diagram](assets/FAB-anatomy.png)`: https://github.com/andrewoverton/doc-material-components/blob/37beee82f99115973dec50423167436afdfd2801/buttons-examples/iOS/ios-FAB.md. Maybe I made it too small though, now that I think about it... And I guess there should be one for each type of FAB right?